CHANGELOG.md

```diff
@@ -1,37 +1,132 @@
-0.0.68
-======
+0.1.0
+=====
+
+This project will begin to follow [SemVer](https://semver.org) in preparation
+for a 1.0.0 release.
 
 New Features
 ------------
 
-- Updated to WebRTC M61 (using [mayeut/libwebrtc](https://github.com/mayeut/libwebrtc))
-- Expanded RTCConfiguration support. We now support `iceTransportPolicy`,
-  `bundlePolicy`, `rtcpMuxPolicy`, and `iceCandidatePoolSize`.
-- Added support for `getConfiguration` and `setConfiguration`.
-- Added support for a non-standard RTCConfiguration attribute, `portRange`. You
-  can use this to constrain the `min` and `max` ports to be used when creating
-  ICE transports. For example,
-
-  ```js
-  const { RTCPeerConnection } = require('wrtc');
-  const pc = new RTCPeerConnection({ portRange: { min: 0, max: 65535 } });
-  ```
-
-  `portRange` cannot be updated by `setConfiguration`.
-- Added support for RTCDataChannel's `id` property.
-- Stubbed out RTCPeerConnection's `canTrickleIceCandidates` property (it always
-  returns `null`).
-- Stubbed out RTCPeerConnection's `connectionState` property (it always returns
-  "new").
-- Added support for RTCPeerConnection's `currentLocalDescription`,
-  `currentRemoteDescription`, `pendingLocalDescription`, and
-  `pendingRemoteDescription` properties.
+Besides updating to WebRTC M61 (using
```

Comment:
According to description in https://github.com/mayeut/libwebrtc/releases/tag/v1.1.1 it is only M60.
Also link in readme is not updated and points to original https://github.com/aisouard/libwebrtc, maybe worth changing it too.

Reply:
Ah crap :-p will update. Thanks for the review! I'm following up with a README.md update soon.

Reply:
I haven't published 0.1.0 yet, only built the binaries. Will update this before `npm publish`.